tcms/static/js/tcms_actions.js

```diff
@@ -1425,19 +1425,7 @@ function exportCase(url, form, table) {
 var printableCases = exportCase;
 
 function validateIssueID(bugSystemId, bugId) {
-  if (bugSystemId == 1) {
-    var result = /^\d{1,7}$/.test(bugId);
-    if (!result) {
-      window.alert(default_messages.alert.invalid_bug_id);
-    }
-    return result;
-  } else if (bugSystemId == 2) {
-    var result = /^[A-Z0-9]+-\d+$/.test(bugId);
-    if (!result) {
-      window.alert(default_messages.alert.invalid_jira_id);
-    }
-    return result;
-  }
+  return true
 }
```

Comment:
Key point of Javascript code related to bug trackers is hardcoded is because the code does not recognize bug trackers dynamically. This is the problem of this function. But, I don't think validateIssueID should always return true though. Issue ID should be validate in the users browser when user just inputs and click Add button. This should be the most difficult part of having the front-end code working with bug trackers defined in database.

While reviewing your patch, I always got invalid id error message whatever valid value I entered for Bugzilla and JIRA.